src/core/Timepicker/style.ts

```diff
@@ -0,0 +1,5 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles(() => ({}));
+
+export { useStyles };
```

Comment:
If there are no styles required we can simply not have a `styles.ts` file instead!